Source/WebCore/bindings/js/SerializedScriptValue.cpp

```diff
@@ -2661,7 +2702,9 @@ class CloneDeserializer : CloneBase {
     CloneDeserializer(JSGlobalObject* lexicalGlobalObject, JSGlobalObject* globalObject, const Vector<RefPtr<MessagePort>>& messagePorts, ArrayBufferContentsArray* arrayBufferContents, Vector<std::optional<ImageBitmapBacking>>&& backingStores, const Vector<uint8_t>& buffer
 #if ENABLE(OFFSCREEN_CANVAS_IN_WORKERS)
         , Vector<std::unique_ptr<DetachedOffscreenCanvas>>&& detachedOffscreenCanvases = { }
+        , const Vector<RefPtr<OffscreenCanvas>>& inMemoryOffscreenCanvases = { }
```

Comment:
I am not familiar with this syntax where we include the types of the members we are initializing. I see it a lot in this file, but I’d like to know more about when we should use this syntax and when we should not.

Reply:
Either I misunderstand your comment or you misread the code. These are construction parameters, not data member initialization.

Reply:
Yes, I misread it.